charts/alertmanager/README.md

```diff
@@ -0,0 +1,9 @@
+# alertmanager
+
+Chart to deploy alertmanager as a statefulset.
+
+## Installation
+
+```sh
```

Comment:
My only thought now is to improve the README a bit. How about the README template laid out for the other charts in this repo?

Reply:
Sounds good! I did not find a template per se but mimicked what I saw in prometheus adapter. PTAL